### DISCUSSION TITLE
syntax error near unexpected token `newline'

### DISCUSSION THREAD

**Discussion by lucascbd** [gravitational/teleport#16132] [Q&A] [answered]
When I try to run the node installation script through the command:
sudo bash -c "$(curl -fsSL https://teleport.example.com/scripts/asdf1234567890asdf0987654321/install-node.sh)"
I get the following message:

bash: -c: line 1: syntax error near unexpected token `newline'
bash: -c: line 1: `<!DOCTYPE html>'

Any thoughs on that?

**Comment by sziyan** [comment]
For those who encounter this error in future, make sure you're teleport instance is not behind any authentication such as Authentik/Authelia.

**Comment by webvictim** [comment, marked as answer]
Theoretically you should be able to work around issues like this by becoming `root` first with `sudo -i` or similar, then just run the command by itself without `sudo`:

```
user@host:~ $ sudo -i
[password]
root@host:~ # curl -fsSL https://teleport.example.com/scripts/asdf1234567890asdf0987654321/install-node.sh
```
